CRM/Core/BAO/CustomField.php

```diff
@@ -1223,7 +1223,18 @@ public static function getDisplayValueCommon(
       case 'AdvMulti-Select':
       case 'Multi-Select':
         if (is_array($value)) {
-          $checkedData = $value;
+          if ($html_type == 'CheckBox') {
+            $newData = array();
+            foreach ($checkedData as $k => $v) {
```

Comment:
@JKingsnorth The build fails due to the undefined variable `$checkedData` used here as the initialisation is removed before the if stmt.